src/totalpave/utilities.h

```diff
@@ -25,6 +25,10 @@
 #include <sqlite3.h>
 
 namespace TP { namespace sqlite {
-    std::string getErrorString(int code);
+
+    constexpr char const * const TOTALPAVE_SQLITE_ERROR_DOMAIN = "com.totalpave.sqlite3.ErrorDomain";
+    constexpr char const * const SQLITE_ERROR_DOMAIN = "com.sqlite3.ErrorDomain";
+    constexpr int const ERROR_CODE_BIND_PARAMETER_ERROR = 1;
+
```

Comment:
The plugin had similar constants but will be updated to use these constants instead. Plugin-specific errors will still have its own plugin specific error domain.